.github/workflows/main.yml

```diff
@@ -64,6 +64,7 @@ jobs:
           - {ROS_DISTRO: noetic, AFTER_SCRIPT: 'rosenv rosrun industrial_ci run_travis', ADDITIONAL_DEBS: "ros-noetic-rosbash"}
           - {ROS_DISTRO: foxy, AFTER_SCRIPT: 'rosenv ros2 run industrial_ci run_travis', ADDITIONAL_DEBS: "ros-foxy-ros2run"}
           - {ROS_DISTRO: galactic, TRACE: true}
+          - {ROS_DISTRO: humble, ROS_REPO: testing}
```

Comment:
Please do not set the ROS_REPO here.
testing is the default anyway!

Reply:
Done